assets/src/edit-story/components/popup/utils.js

```diff
@@ -22,17 +22,17 @@ import { Placement } from '.';
 export function getXTransforms(placement) {
   // left & right
   if (placement.startsWith('left')) {
-    return `-100%`;
+    return -1;
```

Comment:
changed these to pure values so it's easy to obtain transforms in other calculations